website/static/js/logFeed.js

```diff
@@ -1,238 +1,245 @@
+                            if (ctrl.pageToGet() !== ctrl.currentPage()) {
+                                return _buildLogUrl(ctrl.node, ctrl.pageToGet());
+                            }
+                        }
+                    });/* jshint ignore:line */
+                    // function defined inside loop
+                }
+            }
+            // two ellipses
+            else {
+                ctrl.paginators().push({
+                    text: '...',
+                    url: function() { }
+                });
+                for (i = parseInt(ctrl.currentPage()) - 1; i <= parseInt(ctrl.currentPage()) + 1; i++) {
+                    ctrl.paginators().push({
+                        text: i + 1,
+                        url: function() {
+                            ctrl.pageToGet(parseInt(this.text));
+                            if (ctrl.pageToGet() !== ctrl.currentPage()) {
+                                return _buildLogUrl(ctrl.node, ctrl.pageToGet());
+                            }
+                        }
+                    });/* jshint ignore:line */
+                    // function defined inside loop
+                }
+                ctrl.paginators().push({
+                    text: '...',
+                    url: function() { }
+                });
+            }
+            // last page
+            ctrl.paginators().push({
+                text: ctrl.totalPages,
+                url: function() {
+                    ctrl.pageToGet(ctrl.totalPages);
+                    if (ctrl.pageToGet() !== ctrl.currentPage()) {
+                        return _buildLogUrl(ctrl.node, ctrl.pageToGet());
+                    }
+                }
+            });
+            // next page
+            ctrl.paginators().push({
+                url: function() { return ctrl.nextPage; },
+                text: '>'
+            });
+        }
 
-////////////////
-// Public API //
-////////////////
+        return m('.db-activity-list.m-t-md', [
 
-var defaults = {
-    /** Selector for the progress bar. */
-    progBar: '#logProgressBar'
-};
+            ctrl.failed ? m('p', 'Unable to retrieve logs at this time.') :
 
+            ctrl.logRequestPending() ?  m('.spinner-loading-wrapper', [
+                m('.logo-spin.logo-lg'),
+                m('p.m-t-sm.fg-load-message', 'Loading logs...')
+            ]) :
 
-var initViewModel = function(self, logs, url){
-    self.logs = createLogs(logs);
-    self.viewModel = new LogsViewModel(self.logs, url);
-    if(url) {
-        self.viewModel.fetchResults();
+            [ctrl.activityLogs() ? ctrl.activityLogs().map(function(item) {
+                var image = m('i.fa.fa-desktop');
+                if (ctrl.node.anonymous) { item.anonymous = true; }
+                if (!item.anonymous && item.embeds.user && item.embeds.user.data) {
+                    image = m('img', { src : item.embeds.user.data.links.profile_image});
```

Comment:
The image is being requested at the default size (40x40px, I believe), which is much larger than necessary. Better to request a smaller image (16x16) from Gravatar. Check out the Gravatar docs.

I realize this is an issue introduced by the myProjects, so this isn't a blocking change. But I'd suggest making it, since it should be a very small change.